content/api/metrics.apib

```diff
@@ -1601,6 +1642,30 @@ Provides aggregate count of deliveries grouped by the attempt number. This endpo
             ]
         }
 
+### Subject Campaign Metrics [GET /v1/metrics/deliverability/subject-campaign{?from,to,delimiter,domains,campaigns,templates,sending_ips,ip_pools,sending_domains,subaccounts,timezone,limit}]
+
+Provides subject campaign metrics.
+
++ Parameters
+    + from (required, datetime, `2018-07-11T08:00`) ... Datetime in format of `YYYY-MM-DDTHH:MM`.
```

Comment:
`subject_campaigns` can be added as a new parameter here and other aggregate routes